### ISSUE TITLE
Russian speech

### ISSUE THREAD

**Issue by AntonDreamer** [Kyubyong/dc_tts#92]
Is it enough for the Russian voice to edit the hparams and the Russian Voice dataset?

**Comment by simsax** [comment]
Yes, I did a very similar thing  creating an Italian voice. Check out my fork if you want, I made a step by step guide.

**Comment by AntonDreamer** [comment]
> Yes, I did a very similar thing creating an Italian voice. Check out my fork if you want, I made a step by step guide.

Maybe you know how to create transcript.csv automatically? Because I have big data of russian speech and if do this manually it takes more time.

**Comment by Vrystick** [comment]
> Yes, I did a very similar thing creating an Italian voice. Check out my fork if you want, I made a step by step guide.

Hi, in your fork you have added both the English and Italian pre-trained model, but is it necessary to change something in the code(in addition to the path of files specified in your readme) to make it work in Italian too? Because Italian language has also accented characters differently from english.
